modules/ROOT/pages/remove-client-app-latest-task.adoc

```diff
@@ -1,12 +1,28 @@
-= Removing API instance Contracts on API Manager
+= Revoke and Delete API Contracts
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-As Organization Administrator of the master organization, to delete an application that you gave access to your API:
+As the Organization Administrator of the master organization, you can revoke or delete a contract granting an application access to your API. When you revoke a contract, you can restore the contract at any time. 
```

Comment:
> you can revoke or delete a contract granting an application access to your API
This may be misleading. If you are Revoking and Deleting the contract it is not granting access to the API, it is blocking it's access.

Reply:
As the Organization Administrator of the master organization, you can revoke or delete a contract that is granting an application access to your API. When you revoke a contract, you can restore the contract at any time.

Reply:
As the Organization Administrator of the master organization, you can revoke or delete a contract that is providing an application access to your API. When you revoke a contract, you can restore the contract at any time.